cli/flox/src/commands/activate.rs

```diff
@@ -243,6 +233,15 @@ impl Activate {
             ),
             ("FLOX_PROMPT_COLOR_1", prompt_color_1),
             ("FLOX_PROMPT_COLOR_2", prompt_color_2),
+            // Set `FLOX_PROMPT_ENVIRONMENTS` to either the constructed prompt string
+            // or "" if the prompt is disabled.
+            //
+            // The latter is necessary for inner activations to be able to hide the prompt
+            // if the parent activation has it disabled.
+            (
+                FLOX_PROMPT_ENVIRONMENTS_VAR,
+                flox_prompt_environments.unwrap_or_default(),
```

Comment:
hmm I'm thinking, disabling prompts by clearing the prompt variable might be in the way of someone disabling the flox provided prompt AND manually configuring startship (or equivalences) to show the prompt.
I guess we can recover from that with `flox envs prompt` or something like that, but today clearing the prompt var seems like a regression.

Reply:
Hmm that's a good point, I guess we can change that later if it's a problem